packages/dashboard/src/components/pages/home/home.tsx

```diff
@@ -1,17 +1,193 @@
+                            DTrader
+                        </Text>
+                    ),
+                },
+                {
+                    label: 'DBot',
+                    value: () => (
+                        <Text>
+                            DBot
+                        </Text>
+                    ),
+                },
+                {
+                    label: 'SmartTrader',
+                    value: () => (
+                        <Text>
+                            SmartTrader
+                        </Text>
+                    ),
+                },
+                {
+                    label: 'Binary Bot',
+                    value: () => (
+                        <Text>
+                            Binary Bot
+                        </Text>
+                    ),
+                },
+            ],
+        },
+        {
+            default: false,
+            icon: 'IcWalletTradeTypes',
+            label: 'Trade Types',
+            subroutes: [
+                {
+                    label: 'CFDs',
+                    value: () => (
+                        <Text>
+                            CFDs
+                        </Text>
+                    ),
+                },
+                {
+                    label: 'Multipliers',
+                    value: () => (
+                        <Text>
+                            Multipliers
+                        </Text>
+                    ),
+                },
+                {
+                    label: 'Options',
+                    value: () => (
+                        <Text>
+                            Options
+                        </Text>
+                    ),
+                },
+            ],
+        },
+        {
+            default: false,
+            icon: 'IcWalletMarkets',
+            label: 'Markets',
+            subroutes: [
+                {
+                    label: 'Forex',
+                    value: () => (
+                        <Text>
+                            Forex
+                        </Text>
+                    ),
+                },
+                {
+                    label: 'Synthetic Indices',
+                    value: () => (
+                        <Text>
+                            Synthetic Indices
+                        </Text>
+                    ),
+                },
+                {
+                    label: 'Stock Indices',
+                    value: () => (
+                        <Text>
+                            Stock Indices
+                        </Text>
+                    ),
+                },
+                {
+                    label: 'Commodities',
+                    value: () => (
+                        <Text>
+                            Commodities
+                        </Text>
+                    ),
+                },
+            ],
+        },
+    ];
+
+    const list_groups:any[] = [];
+    const subroutes:any[] = [];
```

Comment:
Can we define types for these rather than using `any`?

Reply:
Sure, I used custom types for this